### ISSUE TITLE
New Mexican phones falling validation

### ISSUE THREAD

**Issue by Eliada** [daddyz/phonelib#150]
Hi,

Recently Mexico has added the `56` local code, so phone numbers like this: "5610678471" are valid, but the gem validation fails:
```ruby
[1] pry(main)> require 'phonelib'
=> true
[2] pry(main)> Phonelib.valid_for_country? "5610678471", "MX"
=> false
[3] pry(main)> Phonelib.valid? "+525610678471"
=> false
```
Google [libphonenumber](https://github.com/googlei18n/libphonenumber) has recently updated their data files https://github.com/googlei18n/libphonenumber/commit/ea6d594a385660528317e90dbfce8e508b1b6f2a, so I believe that updating the gem's one might fix the issue.

To summarize, two questions:

1. Please could the data file be updated?
2. How to update the data file? The README shows how to override the `phone_data`, but I just want to add a new entry. How can I use the data files from [libphonenumber](https://github.com/googlei18n/libphonenumber)  to generate a new `phone_data` for the gem?

Perhaps that was more then two questios :grin: 

Thanks!

**Comment by daddyz** [comment]
@Eliada Hi, I updated the gem's data, usually I am updating the data once in a month and publishing new version. You can just update the gem to v0.6.29.
Anyway there is a rake task that updates in case you want to use gem from your own repo.
